.github/workflows/release.yml

```diff
@@ -16,33 +16,121 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-name: Release Apache Spark
+
+# This workflow is intended for use in forked repositories
+# when manually dispatching this to create an RC.
+# To enable full release functionality, developers should manually configure
+# the following GitHub Secrets in their repository settings:
+#
+# - ASF_USERNAME:
+#     Your Apache Software Foundation (ASF) account ID.
+#
+# - ASF_PASSWORD:
+#     The password associated with your ASF account.
+#
+# - GPG_PRIVATE_KEY:
```

Comment:
This means we need to upload our gpg private key into github?

Reply:
Yeah in forked repository. Has to be put in GitHub Actions' Secrets

Reply:
which should be fine. It's a pretty common practice to put sensitive information in GitHub Actions Secrets to be used in CI.